locale/po/grasswxpy_es.po

```diff
@@ -22328,11 +22328,6 @@ msgstr "etiquetas:"
 #~ msgid "Deactive overwrite"
 #~ msgstr "Desactivar sobreescribir"
 
-#~ msgid "Systematic contiguos"
-#~ msgstr "Sistemático continuo"
-
-#~ msgid "Systematic non contiguos"
-#~ msgstr "Sistemático no continuo"
```

Comment:
no idea what these are, why do you remove them?

Reply:
I tried to explain in the PR description. There are 4 files that have these 5 lines removed. The "wrong" source string was already fixed beforehand, so appears in the file way above, and they already have some translations, that exactly match these 5 lines.

In the Russian file, since it didn't match, I left it as is, to let the translation programs handle it. (We're going behind the backs a little bit).

So, there's nothing wrong for the 4 files that have these 5 lines removed.